### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by fru1tstand**
+ Created base project with Spring under Maven with configuration
+ Added Guava, JUnit, Mockito, and Pebble
+ Added setup and deployment documentation

### CODE REVIEW

File: src/main/java/com/jonathanpli/ExceptionNullApplication.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@SpringBootApplication
+public class ExceptionNullApplication {
+	private static final String DEBUG_PARAMETER = "-debug";
+
+	private static Logger logger;
```
Comment (fru1tstand) [inline src/main/java/com/jonathanpli/ExceptionNullApplication.java:22]: This is a test comment. Can we do better?

**Review by fru1tstand — COMMENTED** [review]
This is a huge CL. In the future, please break them into smaller ones.

File: pom.xml [hunk 2]
```diff
@@ -1,69 +1,124 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.jonthanpli</groupId>
-	<artifactId>exceptionnull</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<packaging>jar</packaging>
+    <groupId>com.jonthanpli</groupId>
```
Comment (jonathanpli) [inline pom.xml:7]: Misspelled name

File: pom.xml [hunk 3]
```diff
@@ -1,69 +1,124 @@
...
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.2.RELEASE</version>
-		<relativePath/> <!-- lookup parent from repository -->
-	</parent>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.5.2.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
```
Comment (jonathanpli) [inline pom.xml:19]: Self closing tag

**Review by jonathanpli — CHANGES_REQUESTED** [review]
Have to fix these!

File: pom.xml (hunk 2 above)
Comment (fru1tstand) [reply pom.xml:7]: Fixed in future commit.

File: pom.xml (hunk 3 above)
Comment (fru1tstand) [reply pom.xml:19]: ack.

**Review by jonathanpli — APPROVED** [review]

**Comment by jli227** [comment]
Looks good!

**Pull request merged**